nestkernel/nest_names.cpp

```diff
@@ -73,7 +73,8 @@ const Name b( "b" );
 const Name beta( "beta" );
 const Name beta_Ca( "beta_Ca" );
 const Name buffer_size( "buffer_size" );
-const Name buffer_size_secondary_events( "buffer_size_secondary_events" );
+const Name send_buffer_size_secondary_events( "send_buffer_size_secondary_events" );
+const Name recv_buffer_size_secondary_events( "recv_buffer_size_secondary_events" );
```

Comment:
`send_buffer_size_secondary_events` should be under s and `recv_buffer_size_secondary_events` should be under r.